packages/webpack-plugin/package.json

```
@@ -66,12 +65,17 @@
     "@types/jest": "^28.1.3",
     "@types/node": "^18.6.3",
     "@types/uuid": "^9.0.1",
+    "@types/webpack": "npm:@types/webpack@^4",
     "eslint": "^8.18.0",
     "jest": "^28.1.1",
     "rimraf": "^3.0.2",
     "rollup": "2.75.7",
     "ts-node": "^10.9.1",
-    "typescript": "^4.7.4"
+    "typescript": "^4.7.4",
+    "webpack": "npm:webpack@^4"
+  },
+  "peerDependencies": {
+    "webpack": ">=4"
```

That's a bit too optimistic. How about:

```suggestion
    "webpack": "^4.0.0 || ^5.0.0"
```
?